.appveyor.yml

```
@@ -1,50 +1,50 @@
-- provider: Webhook
-  url: https://app.signpath.io/API/v1/%ORGANIZATION_ID%/Integrations/AppVeyor?ProjectSlug=%PROJECT_SLUG%&SigningPolicySlug=%SIGNING_POLICY_SLUG%&ArtifactConfigurationSlug=%ARTIFACT_CONFIGURATION_SLUG%
-  authorization:
-    secure: nABfe+lksIRfS7lVnkq3EdIYNH8KOka+5ZFTUJqUW2nckBubhrlAXarbCPJHLp9FWWf3zxAWhNwT0It+qyqPRQ==
-
-notifications:
-  - provider: Email
-    to:
-      - gnh1201@catswords.re.kr
-    subject: "Build {{status}}: {{projectName}} {{buildVersion}}"
-    on_build_success: false
+version: 0.2.7.build{build}
```

YAMLlint (1.37.1)

[error] wrong new line character: expected \n

(new-lines)